docs/program/sdks/_index.md

```diff
@@ -75,7 +75,7 @@ You can find more examples of Viam's SDKs in the <file>examples</file> folder of
 
 The easiest way to get started is to navigate to your robot's page on [the Viam app](https://app.viam.com/robots), select the **Code Sample** tab, select your preferred SDK, and copy the code generated for you.
 
-These boilerplate code samples import all of the necessary libraries and set up a client connection to your {{< glossary_tooltip term_id="remote" text="remote">}} or local robot.
+These boilerplate code samples import all of the necessary libraries and set up a client connection to your remote or local robot.
```

Comment:
opt: do we need `remote or local` in the first place here? From the perspective of the Viam app, all robots are remote! Very optional, just a thought.

Reply:
```suggestion
These boilerplate code samples import all of the necessary libraries and set up a client connection to your robot, whether your code runs on your robot or on a different machine.
```